shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -56,12 +56,30 @@
     // Metrics in Segment Store Service
     // Segment-related stats
     public static final String SEGMENT_CREATE_LATENCY = "segmentstore.segment.create_latency_ms"; // Histogram
+    public static final String SEGMENT_DELETE_LATENCY = "segmentstore.segment.delete_latency_ms"; // Histogram
     public static final String SEGMENT_READ_LATENCY = "segmentstore.segment.read_latency_ms";     // Histogram
     public static final String SEGMENT_WRITE_LATENCY = "segmentstore.segment.write_latency_ms";   // Histogram
     public static final String SEGMENT_READ_BYTES = "segmentstore.segment.read_bytes";            // Counter and Per-segment Counter
     public static final String SEGMENT_WRITE_BYTES = "segmentstore.segment.write_bytes";          // Counter and Per-segment Counter
     public static final String SEGMENT_WRITE_EVENTS = "segmentstore.segment.write_events";        // Counter and Per-segment Counter
 
+    // Table Segment stats
+    public static final String TABLE_SEGMENT_UPDATE_LATENCY = "segmentstore.tablesegment.update_latency_ms";
```

Comment:
To be consistent with the other metric names in this class, could you please add a comment to the end of each new line stating the metric type?

Reply:
Ok.